go.mod

```diff
@@ -22,22 +22,26 @@ require (
 	sigs.k8s.io/controller-runtime v0.17.2
 )
 
+require (
+	github.com/Masterminds/semver/v3 v3.2.1 // indirect
+	github.com/go-jose/go-jose/v4 v4.0.1 // indirect
+)
```

Comment:
pls move this require block in the existing require block

Reply:
Done